pkg/cmdutil/flags.go

```diff
@@ -120,3 +123,39 @@ func (e *enumValue) String() string {
 func (e *enumValue) Type() string {
 	return "string"
 }
+
+type enumMultiValue struct {
+	value   *[]string
+	options []string
+}
+
+func (e *enumMultiValue) Set(value string) error {
+	items := strings.Split(value, ",")
+	for _, item := range items {
+		if !isIncluded(item, e.options) {
+			return fmt.Errorf("valid values are %s", formatValuesForUsageDocs(e.options))
+		}
+	}
+	*e.value = items
```

Comment:
The existing slice of values should be only appended to after each Set invocation, not replaced. Note that this will result in 3 separate Set invocations:
```
--in name --in readme --in description
```

Reply:
Ahh, good catch, had not thought about that way of invocation.